src/Microsoft.Azure.ServiceBus/Primitives/EntityNameHelper.cs

```diff
@@ -8,11 +8,13 @@ namespace Microsoft.Azure.ServiceBus
     /// </summary>
     public static class EntityNameHelper
     {
-        private const string PathDelimiter = @"/";
-        private const string Subscriptions = "Subscriptions";
-        private const string SubQueuePrefix = "$";
-        private const string DeadLetterQueueSuffix = "DeadLetterQueue";
-        private const string DeadLetterQueueName = SubQueuePrefix + DeadLetterQueueSuffix;
+        public const string PathDelimiter = @"/";
```

Comment:
They don't, I made them private in my previous PR: https://github.com/Azure/azure-service-bus-dotnet/pull/137/files#diff-96579f4f545d4098216eff74b430e8f2